doc/registry_api.md

```diff
@@ -0,0 +1,21 @@
+# Registry Api
```

Comment:
Add this doc here https://github.com/namecheap/ilc-sdk/blob/master/typedoc.json#L9

Reply:
done